sdk/keyvault/pom.service.xml

```diff
@@ -9,22 +9,10 @@
   <packaging>pom</packaging>
   <version>1.0.0</version><!-- Need not change for every release-->
   <modules>
-
-    <!-- Track 1 -->
```

Comment:
What is the plan to re-enable track 1?

Reply:
We discussed a temporary convention recently. Let's see if we can capture it here:

1. Introduce a ```ci.data.yml``` file which sits along side the existing ```ci.yml``` file.
2. This file would turn on Jetty and specify the different set of artifacts.
3. Introduce profiles into ```pom.service.xml``` so that we can build the track 1 libraries or the track 2 libraries through the same aggregate.
4. Create a pipeline called ```java - [service] - ci.data``` (public) and ```java - [service] - data``` (internal).

Reply:
This would only apply when:

1. We have track 1 and track 2 libraries where we want to/need to seperate them.
2. Can't deprecate the need to build and release the track 1 libraries.

The preference will be to:

1. Co exist in the same pipeline
2. Disable track 1 builds if we don't need to build them anymore.
3. Setup a pipeline as described above.

Longer term we'll be removing these track 1 pipelines.

Reply:
OK in this case then given both set of code still exists in the repo I'd go ahead and setup the track 1 CI as well so that we don't have folks submitting changes to it without any validation.